addons/website_forum/views/website_forum.xml

```diff
@@ -145,7 +145,7 @@
                             </div>
                         </t>
                     </div>
-                    <div class="card">
```

Comment:
Could you use `mt-2` or `mt-3` on the bottom cards instead ?

Reply:
Oky :+1: